src/main.cpp

```diff
@@ -339,6 +339,63 @@ auto init_irods_connection_pool(const json& _config) -> irods::connection_pool
     };
 } // init_irods_connection_pool
 
+auto load_oidc_configuration(const json& _config) -> void
+{
+  nlohmann::json oi_config;
+        nlohmann::json endpoint_config;
+
+        // Scope as we're likely going to move this part out into a sep func...
+        try {
+			// Setup config
+            oi_config = _config.at(json::json_pointer{"/authentication/oidc"});
```

Comment:
indentation

Reply:
Deferring formatting discussions to this thread(?): https://github.com/irods/irods_client_http_api/pull/75#discussion_r1334508464

Reply:
Feel free to mark this and others like as resolved if linked to the same comment.